src/pcapi/core/search/backends/appsearch.py

```diff
@@ -70,6 +75,28 @@
     {"tome", "T."},
 )
 
+VENUES_ENGINE_NAME = "venues"
+
+VENUES_SCHEMA = {
+    "name": "text",
+    "venueType": "text",
```

Comment:
Les noms de champ sont en snake_case dans App Search : 

```suggestion
    "venue_type": "text",
```

idem pour les autres champs plus bas.